exampleCourse/questions/workshop/Lesson3_example4_v2/question.html

```diff
@@ -10,7 +10,7 @@
       <pl-location left="20" top="15">  <span class="bg-white p-2 border">{{params.x}}</span> </pl-location>
       <pl-location left="20" top="65">  <span class="bg-white p-2 border">{{params.y}}</span> </pl-location>
       <pl-location left="20" top="105"> <span class="bg-white p-2 border">{{params.z}}</span> </pl-location>
-      <pl-location left="365" top="65"> <pl-integer-input answers-name="F" size=15 show-help-text="false"  ></pl-integer-input> </pl-location>
+      <pl-location left="365" top="65"> <pl-integer-input aria-label="Output value" answers-name="F" size=15 show-help-text="false"  ></pl-integer-input> </pl-location>
```

Comment:
Another case where I feel like we're not well-served by just trying to satisfy an automated checker. Sure, this is probably marginally better than were things were before, but I don't want us to fool ourselves into thinking that this question is accessible after this change.

Reply:
This specific question would benefit from alt-text on the image that correlates to the `aria-label`s. I also don't think this question is accessible after this change.

Reply:
Yep, alt-text would definitely help. One interesting thing I noticed while looking at this: `<pl-overlay>` puts the image after the overlayed things (inputs, etc.), so someone moving through the document in order would discover the inputs before the image that would provide the necessary context about what the inputs are and how they're related.

<img width="659" alt="Screenshot 2025-04-30 at 13 11 21" src="https://github.com/user-attachments/assets/c77ae459-2322-4f91-b1cf-aeb7dd2e73f1" />

Nothing to do about this in this PR, but maybe worth doing something about when we decide to resolve the alt-text issues.